### PR TITLE
Add initial spinner

### PR DESCRIPTION
**Pull request by PaulioRandall**
**What does this change?**

Adds a rudimentary spinner. To be replaced by a designed spinner later.

**Why?**

Gotta have a spinner to tell the user when stuff is loading.

**How?**

SVG as a Svelte component with a few props to set classes on the parts of the spinner.

Uses SVG animation rather than CSS animations. This means the code may be easily copied into an `.svg`.

**Does this introduce new dependencies?**

No.

**How is it tested?**

Stories.

**How is it documented?**

Stories.

**Are light and dark themes considered?**

Not yet.

**Is it complete?**
- [x] Have you included changeset file?
- [ x] If this adds a new component, is it exported via `index.js`?

### CODE REVIEW

File: packages/ui/src/lib/spinners/Spinner.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	<div class="text-core-grey-800">
+		<Spinner class="w-40 stroke-[16]" />
+	</div>
+</Story>
+
+<Story name="Color">
+	<div class="text-core-grey-800">
+		<Spinner class="w-40" circleClass="stroke-core-blue-200" arcClass="stroke-core-glaPink-600" />
+	</div>
+</Story>
```
Comment (jamesscottbrown) [inline packages/ui/src/lib/spinners/Spinner.stories.svelte:25]: An example  showing the spinner working inside a button might be useful:

```suggestion
</Story>

<Story name="In a button">
	<div class="text-core-grey-800">
		<Button>I'm loading...
			<Spinner class="h-6 w-6"/>
		</Button>
	</div>
</Story>
```

File: packages/ui/src/lib/spinners/Spinner.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<script>
+	import { Meta, Story } from '@storybook/addon-svelte-csf';
+
+	import Spinner from './Spinner.svelte';
```
Comment (jamesscottbrown) [inline packages/ui/src/lib/spinners/Spinner.stories.svelte:4]:
```suggestion
	import Spinner from './Spinner.svelte';
	import Button from "../button/Button.svelte";
```

This import would be needed if you accept my suggestion below.

File: packages/ui/src/lib/index.js [hunk 3]
```diff
@@ -13,9 +13,10 @@ export { default as RadioButtonGroup } from './radioButton/RadioButtonGroup.svel
 export { default as RadioButtonSolid } from './radioButtonSolid/RadioButtonSolid.svelte';
 export { default as RadioButtonGroupSolid } from './radioButtonSolid/RadioButtonGroupSolid.svelte';
 export { default as Select } from './select/Select.svelte';
+export { default as Spinner } from './spinners/Spinner.svelte'
```
Comment (jamesscottbrown) [inline packages/ui/src/lib/index.js:16]:
```suggestion
export { default as Spinner } from './spinners/Spinner.svelte';
```

Prettier wants a semi-colon here.

File: packages/ui/src/lib/index.js [hunk 4]
```diff
@@ -13,9 +13,10 @@ export { default as RadioButtonGroup } from './radioButton/RadioButtonGroup.svel
...
 export { default as TabbedSidebar } from './tabbedSidebar/TabbedSidebar.svelte';
 export { default as TabbedSidebarTabLabel } from './tabbedSidebar/TabbedSidebarTabLabel.svelte';
 export { default as TabbedSidebarTabList } from './tabbedSidebar/TabbedSidebarTabList.svelte';
 export { default as TabbedSidebarWrapper } from './tabbedSidebar/TabbedSidebarWrapper.svelte';
 export { default as TabLabel } from './tabs/TabLabel.svelte';
-export { default as TabList } from './tabs/TabList.svelte';
+export { default as TabList } from './tabs/TabList.svelte';
```
Comment (jamesscottbrown) [inline packages/ui/src/lib/index.js:22]:
```suggestion
export { default as TabList } from './tabs/TabList.svelte';

```

Prettier wants a blank line at the end of this file.

**Review by jamesscottbrown — COMMENTED** [review]
The implementation of this looks fine.

It needs a changeset file with a message like `ADDED: Spinner component` though, so that this gets automatically added to the CHANGELOG on the next release.

From a design POV I'm not sure about the `stroke-linecap="round"`, as most of our new design system is pretty square. @ChrisKnightLDN can give an opinion when he's back, but I think it could be merged and then visually tweaked later.

**Comment by ChrisKnightLDN** [comment]
The design should be as per the component in Figma. Although it's accepted as a bit of a wip - where there is a component, as there is with the loading spinner it should match that visually. We can also keep design focussed discussions in Figma. **TLDR. square linecap**

**Pull request merged**